src/design-system/tokens.scss

```diff
@@ -1,6 +1,6 @@
 :root {
   // Icons
```

Comment:
is this needed?

Reply:
I see @mnavarroespinosa comment and I agree then, we shouldn't use values that are not expected in XDS in the archetype. So I will remove this. Thks!